site/content/components/getPageBody.jsx

```diff
@@ -1,19 +1,27 @@
-import { useMDXComponent } from "next-contentlayer/hooks";
```

Comment:
As I said, if this logic is really needed, let's at least move it to the getter function.